src/Backend/Modules/Pages/Engine/Model.php

```diff
@@ -1430,7 +1430,7 @@ public static function updateRevisionData(int $pageId, int $revisionId, array $d
             [$pageId, $revisionId]
         );
     }
-
+    
```

Comment:
you have some whitespace remaining

Reply:
Strange, don't see it in my editor :|

![Schermafbeelding 2020-06-01 om 16 20 34](https://user-images.githubusercontent.com/4540274/83418412-d7d39600-a423-11ea-83df-bb07586ceb90.png)